src/webpack/index.ts

```diff
@@ -98,6 +98,25 @@ export class ReactLooselyLazyPlugin {
 
       callback();
     });
+
+    compiler.hooks.thisCompilation.tap(name, compilation => {
+      // modifies the webpack bootstrap code generated to expose jsonpScriptSrc
+      // only needed on Webpack 4.x as Webpack 5+ has official support.
```

Comment:
This is actually no longer true since I've also exposed `installedChunks` 😅 
Not sure what happens with webpack 5 and whenever this actually works with it